app/controllers/pages_controller.rb

```diff
@@ -1,6 +1,7 @@
 class PagesController < ApplicationController
   def home
-    @tools = Tool.order('reproducibility_score DESC').limit(100).all
+    @tools = Tool.order('reproducibility_score DESC')
+                 .page(params[:page]).per_page(25)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.